README.md

```diff
@@ -19,83 +19,76 @@ page](https://github.com/shawnlaffan/biodiverse).
-perl project](https://strawberryperl.com/releases.html), and there is
-a function to do this for you. Note that this will take a while and
-send large amounts of text to the console.  
-
-```
-devtools::load_all()
-BiodiverseR::install_strawberry_perl()
-```
+perl project](https://strawberryperl.com/releases.html). This will be
+downloaded automatically when using the commands below.
 
-### Unix/Mac
+Most unix-derived systems such as Linux and Mac provide a perl
+interpreter but it is best to avoid this and install
+[perlbrew](https://perlbrew.pl/Installation.html) so you have a separate
+installation.  
+When you install perlbrew be sure to also install the cpanm utility (run
+this in your shell after starting perlbrew: `perlbrew install-cpanm`).
 
-Most unix-derived systems provide a perl interpreter but it is best to avoid
-this and use a system like [perlbrew](https://perlbrew.pl/). Be sure to
-also install the cpanm utility (see perlbrew site for details).
+You also need to have git installed on your system.
 
-Once you have a perl installed and in your path you can install the perl
-dependencies using cpanm at the command line. (Make sure to update the
-below code to use the correct path separator on Windows). Note that this
-requires that you have git installed on your system and in the path.
+2.  Install the R code
 
-This code also assumes you have run a git clone of this repo and are at
-the top level of this repo.
+You can install the R code like so:
 
-``` bash
-cpanm https://github.com/shawnlaffan/biodiverse.git
-cd inst/perl
-cpanm --installdeps .
+``` r
+# install.packages("devtools")
+library("devtools")
+install_github("shawnlaffan/BiodiverseR")
 ```
 
-Note that the above will take a while if you do not already have the
-GDAL development package installed on your system. This is because it
-will compile its own version if it is unable to find one on the system
-(but maybe this is not such a bad thing as then it will be isolated from
-system changes).
+However, it is currently best to work within the git repo given ongoing
+development updates.
 
-If you want to see things as they happen then add the verbose flag to
-the cpanm calls (`cpanm --verbose ...`).
-
-2.  Install the R code
-
-(You might already have done this in one of the steps above).
-
-You can install the R code like so:
+Set your working directory to be the top of the git repo and then run
+this:
 
 ``` r
-devtools::install_github("shawnlaffan/BiodiverseR")
+# install.packages("devtools")
+library("devtools")
+load_all()
 ```
 
-However, it is currently best to work within the git repo.
+These next commands will install the Biodiverse engine and its perl
+dependencies. The first one does nothing on Windows but there is no harm
+in running it.
 
 ``` r
-devtools::load_all()
+init_perlbrewr()
+install_perl_deps()
 ```
 
+Note that the above will take a while if you do not already have the
+GDAL development package installed on your system. This is because it
```

Comment:
Maybe instructions on how to get the GDAL package could be helpful, otherwise a link to GDAL's documentation might be enough.

Although, simplicity is important, so maybe adding that would make the README more convoluted, especially for someone with a non-technical background.

Reply:
Good point.  We could link to https://gdal.org/en/latest/download.html#binaries

Ultimately non-dev users will not need to do this as we will have binaries available for the main operating systems.  Until then, though...

Reply:
Done in 5240499429969d983bc3e54dd3eab10dc1689d77